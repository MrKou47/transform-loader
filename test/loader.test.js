/* eslint-disable
  prefer-destructuring,
*/
import webpack from './helpers/compiler';

describe('Loader', () => {
  test('Defaults', async () => {
    const config = {
      module: {
        rules: [{
          test: /\.coffee?$/,
          loader: `${__dirname}/../src/cjs?coffeeify`,
        }],
      },
    };

    const stats = await webpack('fixture.js', config);
    const { source } = stats.toJson().modules[1];

    expect(source).toMatchSnapshot();
  });
});
