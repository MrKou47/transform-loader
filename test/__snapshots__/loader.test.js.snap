// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Defaults 1`] = `
"ua = navigator.userAgent
ua = ua.toLowerCase()
module.exports = ua;"
`;
